WebService/Runtime/Config.cs

```diff
@@ -267,6 +268,7 @@ private static IServicesConfig GetServicesConfig(IConfigData configData)
                 SimulationsStorage = GetStorageConfig(configData, SIMULATIONS_STORAGE_KEY),
                 DevicesStorage = GetStorageConfig(configData, DEVICES_STORAGE_KEY),
                 PartitionsStorage = GetStorageConfig(configData, PARTITIONS_STORAGE_KEY),
+                UserAgent = configData.GetString(USER_AGENT_KEY),
```

Comment:
Why doesn't this line call GetStorageConfig?

Reply:
because user agent has got nothing to do with StorageConfig. I see GetStorageConfig returning values for all storage related config information. User Agent is a string to identify the application and so, I felt it din't add value by putting it in the GetStorageConfig function.

Reply:
we are reading a plain string value from config as in line 259, 260